Software/app/basic_lorawan/README.md

```diff
@@ -0,0 +1,59 @@
+# basic\_lorawan
+
+## Description
+
+`basic_lorawan` serves as a basic template for LoRaWAN applications.
+
+## Configuration
+
+### Default
+
+1. Follow [this](https://www.thethingsindustries.com/docs/devices/adding-devices/) guide to add your device to TTN.
+2. Change `APPEUI`, `DEVEUI` and `APPKEY` in [conf/app_conf.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/basic_lorawan/conf/app_conf.h) to the values for you specific device on TTN.
+3. Uncomment the REGION parameters in [conf/lorawan_conf.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/basic_lorawan/conf/lorawan_conf.h) and set `ACTIVE_REGION` in [lora_app.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/basic_lorawan/lora_app.h). These should correspond to the location that you use these devices for.
+
+### Low power
+
+1. Make sure that `LOW_POWER_DISABLE` is kept at 0 in [conf/app_conf.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/basic_lorawan/conf/app_conf.h).
+2. Set `GNSE_ADVANCED_TRACER_ENALBE` to 0 in [conf/app_conf.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/basic_lorawan/conf/app_conf.h). This will make the device consume less power, but will also disable UART.
+
+Additionally, try to remember that:
+- Setting the debugger will consume more power. The debugger is set in the macro `DEBUGGER_ON` in [conf/app_conf.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/basic_lorawan/conf/app_conf.h).
+- Keeping peripherals, such as the sensors, off by controlling the load switches will reduce power consumption significantly.
+- The less transmissions over time, the lower the power consumption will be. This is set by `APP_TX_DUTYCYCLE` in [lora_app.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/basic_lorawan/lora_app.h).
+
+### LoRa extra's
+
+Some extra setup configurations can be done to change the behaviour of the LoRaWAN capabilities. These include:
+1. Setting the activation method (OTAA or ABP) in `LORAWAN_DEFAULT_ACTIVATION_TYPE` in [lora_app.h](https://github.com/TheThingsIndustries/generic-node-se/blob/develop/Software/app/basic_lorawan/lora_app.h). OTAA is recommended as it is the most secure.
```

Comment:
Wrap lora_app.h as `lora_app.h` and repeat for all occurrences

Reply:
You can also link this: https://www.thethingsindustries.com/docs/devices/abp-vs-otaa/